README.md

```diff
@@ -21,7 +21,7 @@
 
 [Fedimint](https://fedimint.org) is a module based system for building federated applications. It is designed to be a trust-minimized, censorship-resistant, and private alternative to centralized applications.
 
-> **Fedimint is alpha software released under an [MIT License](https://github.com/fedimint/fedimint/blob/master/LICENSE) and recently cut its 0.1 release. We recommend you run Fedimint on testnets like mutinynet, or on mainnet with small amounts of money.**
+> **Fedimint is alpha software released under an [MIT License](https://github.com/fedimint/fedimint/blob/master/LICENSE) and recently cut its [0.1 release](https://github.com/fedimint/fedimint/releases/tag/v0.1.3). We recommend you run Fedimint on testnets like mutinynet, or on mainnet with small amounts of money.**
```

Comment:
You could just link to https://github.com/fedimint/fedimint/releases/latest, so the most recent release is always behind this link

Reply:
updated